mmcv/ops/deform_conv.py

```diff
@@ -46,6 +46,71 @@ def symbolic(g,
             bias_i=bias,
             im2col_step_i=im2col_step)
 
+    @staticmethod
+    def _calculate_npu_sort_index(kernel_h, kernel_w, deformable_group):
+        split_num = deformable_group * 2 * kernel_h * kernel_w
+        sort_index_for_npu = list(range(split_num))
+        sort_index_for_npu_fp = (
+            sort_index_for_npu[1::2] + sort_index_for_npu[::2])
+        sort_index_for_npu_bp_dict = {
+            i: idx
+            for idx, i in enumerate(sort_index_for_npu)
+        }
+        sort_index_for_npu_bp = [
```

Comment:
Since `sort_index_for_npu = list(range(split_num))`
`sort_index_for_npu_bp_dict` is always `{0:0, 1:1, ... }` and `sort_index_for_npu_bp == sort_index_for_npu`.
It this expected?

Reply:
Sorry, it is a mistake. I fix this static function of dcnv2 , and directly reference it here.